src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -1922,8 +1922,16 @@ HRESULT CEndlessUsbToolDlg::OnInstallDualBootClicked(IHTMLElement* pElement)
 
 	if (UpdateDualBootTexts()) {
 		m_selectedInstallMethod = InstallMethod_t::UninstallEndless;
+		eventTracking(_T("Selected"));
```

Comment:
As you said on Slack, a Selected event for UninstallEndless should be logged from `endless-uninstaller.exe` too.